lib/modules/asset/AssetService.ts

```diff
@@ -306,6 +406,45 @@ export class AssetService {
     return result;
   }
 
+  /**
+   * Replace an asset metadata
+   */
+  public async mReplace(
```

Comment:
Nit: from the behavior this function is replacing and historizing no?

Reply:
That's what it does indeed